src/ScottPlot5/ScottPlot5/Primitives/Label.cs

```diff
@@ -77,6 +77,19 @@ public void SetBestFont()
         FontName = Fonts.Detect(Text);
     }
 
+    private SKTypeface? CachedTypeface = null;
+
+    private SKTypeface Typeface
+    {
+        get
+        {
+            if (CachedTypeface is null)
+            {
+                CachedTypeface = Fonts.GetTypeface(FontName, Bold, Italic);
+            }
+            return CachedTypeface;
+        }
+    }

```

Comment:
Forgive me if I'm missing something, but this isn't used anywhere is it? I believe #3841 changed `Label` such that this no longer does anything.

I also believe #3841 is also supposed to have addressed the performance concerns? I believe I tested it when it came out, but I'd appreciate if you could confirm that it helped? Unfortunately I tested it under WSL, which isn't ideal for making performance conclusions.